### DISCUSSION TITLE
network/modules/spiderpool/userdma/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3379] [Announcements]
# network/modules/spiderpool/userdma/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/network/modules/spiderpool/userdma/

**Comment by panpan0000** [comment]
2个问题：

1.一个是如果RDMA选的是macvlan，而不是SRIOV， 则“网络资源参数”处下拉框是不同的。解释下，跟https://docs.daocloud.io/network/modules/spiderpool/rdmapara/#macvlan-ipvlan-roce页面里的rdmaSharedDevicePlugin.deviceConfig.resourceName	一致

2."运行命令：添加如下内容:" 需要解释一下。然后镜像用了rping-test，也应该用一下rping命令。
